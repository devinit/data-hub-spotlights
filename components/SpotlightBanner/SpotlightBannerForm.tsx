import React, { FunctionComponent } from 'react';

const SpotlightBannerForm: FunctionComponent = ({ children }) => {
  return (
    <div className="spotlight-banner__form">{ children }</div>
  );
};

export { SpotlightBannerForm };
