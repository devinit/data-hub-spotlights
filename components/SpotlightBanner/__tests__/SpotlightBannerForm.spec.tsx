/**
 * @jest-environment jsdom
 */
import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import { SpotlightBannerForm } from '../SpotlightBannerForm';

describe('SpotlightBannerForm', () => {
  test('renders correctly', () => {
    const renderer = TestRenderer.create(<SpotlightBannerForm/>).toJSON();

    expect(renderer).toMatchSnapshot();
  });

  test('renders children correctly', () => {
    const renderer = TestRenderer.create(
      <SpotlightBannerForm>
        <div>My Child</div>
      </SpotlightBannerForm>).toJSON();

    expect(renderer).toMatchSnapshot();
  });
});
