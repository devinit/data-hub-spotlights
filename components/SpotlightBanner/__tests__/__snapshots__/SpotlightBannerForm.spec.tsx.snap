// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpotlightBannerForm renders children correctly 1`] = `
<div
  className="spotlight-banner__form"
>
  <div>
    My Child
  </div>
</div>
`;

exports[`SpotlightBannerForm renders correctly 1`] = `
<div
  className="spotlight-banner__form"
/>
`;
